core/transcoder.go

```diff
@@ -131,32 +146,30 @@ func (n *LivepeerNode) getSegmentChan(job *ethTypes.Job) (SegmentChan, error) {
 	return sc, nil
 }
 
-func (n *LivepeerNode) TranscodeSegment(job *ethTypes.Job, ss *SignedSegment) error {
+func (n *LivepeerNode) TranscodeSegment(job *ethTypes.Job, ss *SignedSegment) (*TranscodeResult, error) {
 	glog.V(common.DEBUG).Infof("Starting to transcode segment %v", ss.Seg.SeqNo)
 	ch, err := n.getSegmentChan(job)
 	if err != nil {
 		glog.Error("Could not find segment chan ", err)
-		return err
+		return nil, err
 	}
-	segChan := &SegChanData{seg: ss, res: make(chan error, 1)}
+	segChan := &SegChanData{seg: ss, res: make(chan *TranscodeResult, 1)}
```

Comment:
Small thing - `segChan` seems like a misnomer here.  Should be `segChanData`?

Reply:
Fixed https://github.com/livepeer/go-livepeer/pull/512/commits/25e7e8b85b9038aee67a242f037d9794cf26d9d0